editor.planx.uk/src/@planx/components/FindProperty/Public/index.tsx

```diff
@@ -204,6 +213,14 @@ function Component(props: Props) {
           if (regions) {
             newPassportData["property.region"] = regions;
           }
+          if (titleBoundary) {
+            const areaSquareMetres =
+              Math.round(area(titleBoundary as FeatureCollection) * 100) / 100;
+            newPassportData["property.boundary.title"] = titleBoundary;
+            newPassportData["property.boundary.title.area"] = areaSquareMetres;
+            newPassportData["property.boundary.title.area.hectares"] =
+              areaSquareMetres / 10000;
```

Comment:
These passport variable names and area formatting are consistent with draw calculations handled here https://github.com/theopensystemslab/map/blob/45fa7284c7b0f3d15b2aadb69d64807d42749c77/src/components/my-map/utils.ts#L21

Reply:
nit: See comment above about needing an alternate way to calculate area - ideally it would be nice if this could be handled just once.

Currently it's the responsibility of the map component for `property.boundary.area` but PlanX for `property.boundary.title.area` which feels a bit inconsistent.

Reply:
Conversion from sq m to ha now added to utils as a helper function!